src/ric/d_18C40.c

```diff
@@ -0,0 +1,330 @@
+#include "ric.h"
+
+u8 D_80154C40[] = {0x30, 0x3F, 0x20, 0x2F, 0x10, 0x1E, 0x10, 0x3F,
+                   0x1F, 0x1F, 0x30, 0x30, 0x10, 0x2F, 0x00, 0x00};
+
+u16 D_80154C50[] = {0x0000, 0xFFFC, 0xFFF8, 0xFFF4, 0xFFF0, 0xFFEC};
+
+s32 D_80154C5C[] = {0xFFFFD000, 0xFFFFC000, 0xFFFFA000,
+                    0xFFFF8000, 0xFFFF6000, 0xFFFF4000};
+
+u16 D_80154C74[] = {0x0030, 0x0040, 0x0050, 0x0060, 0x0070, 0x0080};
+
+AnimationFrame D_80154C80[] = {
+    0x0001, 0x0001, 0x0001, 0x0002, 0x0001, 0x0003, 0x0001, 0x0004, 0x0001,
+    0x0005, 0x0001, 0x0006, 0x0001, 0x0007, 0x0001, 0x0008, 0x0001, 0x0009,
+    0x0001, 0x000A, 0x0001, 0x000B, 0x0001, 0x000C, 0x0001, 0x000D, 0x0001,
+    0x000E, 0x0001, 0x000F, 0x0001, 0x0010, 0x0001, 0x0011, 0x0001, 0x0012,
+    0x0001, 0x0013, 0x0001, 0x0014, 0x0001, 0x0015, 0x0001, 0x0016, 0x0001,
+    0x0017, 0x0001, 0x0018, 0xFFFF, 0x0000,
+};
+
+u8 D_80154CE4[] = {0x02, 0x09, 0x03, 0x0A, 0x01, 0x08, 0x04, 0x0B,
+                   0x00, 0x07, 0x05, 0x0C, 0x06, 0x0D, 0x00, 0x00};
+
+u8 D_80154CF4[] = {
+    0x02, 0x09, 0x03, 0x0A, 0x04, 0x0B, 0x05, 0x0C, 0x06, 0x0D, 0x00, 0x00,
+    0x08, 0xC0, 0x60, 0x00,
+
+    0x01, 0x01, 0x04, 0x00, 0x33, 0x00, 0x03, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x10, 0x7F, 0x7F, 0x7F,
+
+    0x01, 0x01, 0x02, 0x00, 0x33, 0x00, 0x01, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x08, 0x7F, 0x7F, 0x7F,
+
+    0x02, 0x02, 0x02, 0x00, 0x33, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x06, 0x7F, 0xFF, 0xFF,
+
+    0x01, 0x01, 0x04, 0x00, 0x73, 0x00, 0x03, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x0C, 0xC0, 0x60, 0x00,
+
+    0x01, 0x01, 0x04, 0x00, 0x33, 0x00, 0x03, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x0C, 0x7F, 0x00, 0x00,
+
+    0x03, 0x03, 0x02, 0x00, 0x02, 0x00, 0x04, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x08, 0x1F, 0x1F, 0x7F,
+
+    0x01, 0x01, 0x04, 0x00, 0x33, 0x00, 0x06, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x14, 0x7F, 0x7F, 0xC0,
+
+    0x01, 0x01, 0xFE, 0xFF, 0x33, 0x00, 0x07, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x06, 0xC0, 0xC0, 0xC0,
+
+    0x02, 0x02, 0x02, 0x00, 0x7B, 0x00, 0x08, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x10, 0x7F, 0x7F, 0x7F,
+
+    0x01, 0x01, 0x02, 0x00, 0x33, 0x00, 0x09, 0x00, 0x00, 0x00, 0x80, 0x08};
+
+unkStr_8011E4BC* D_80154DA0[] = {
```

Comment:
I think this one is also wrong, if the first array is size 10, then I think 0xc would be the next offset. So the previous array starting from offset 0xc should be `unkStr_8011E4BC`s